### DISCUSSION TITLE
Getting the total sales per each product

### DISCUSSION THREAD

**Discussion by jackabox** [statamic-rad-pack/shopify#6] [Feature Requests]
We can expand the API here to pull in reports of all sales per product (will be typically slow for the first pull). Then integrate the webhook so that when an order is placed it updates the total sold per variant also. 

This would allow the developer to make a "sort by most popular" option.

Not sure if this would be overly used by anyone so currently have this as a potential feature for post-initial release.

Let me know your thoughs

**Comment by austriker27** [comment]
I would totally use this feature and would love to see it! 

Adding total sales and more data so we could add "trending products" would be super useful. (Not sure how to best determine "trending", maybe most sales in last 30 days?)

**Comment by ryanmitchell** [comment]
https://github.com/statamic-rad-pack/shopify/pull/191
